plotly_express/_core.py

```diff
@@ -597,6 +599,51 @@ def infer_config(args, constructor, trace_patch):
         if param in args and args[param] is None:
             args[param] = getattr(defaults, param)
 
+    # load the default template if set, otherwise "plotly"
+    if args["template"] is None:
+        if pio.templates.default is not None:
+            args["template"] = pio.templates.default
+        else:
+            args["template"] = "plotly"
+
+    # retrieve the actual template if we were given a name
+    try:
+        template = pio.templates[args["template"]]
+    except Exception:
+        template = args["template"]
```

Comment:
Is the idea here to catch a `go.layout.Template` object being passed in directly?

Reply:
Yeah